### DISCUSSION TITLE
can not send request.

### DISCUSSION THREAD

**Discussion by wenxuankou** [darrenburns/posting#149] [Q&A]
<img width="522" alt="image" src="https://github.com/user-attachments/assets/ea3fd717-e81d-44d7-b308-aa8bc0c772a3">

**Comment by wenxuankou** [comment]
when i first open the posting. i try to send a request. but get such a error. i do not know how to resolve it.

i think i had installed it correctly.

<img width="617" alt="image" src="https://github.com/user-attachments/assets/ed7f6713-3651-4292-ab75-45f9d7d6c551">

**Comment by darrenburns** [comment]
Sorry for the delay.

You'd need to share more info on what you filled out the request with for me to be able to help.
